src/guc.c

```diff
@@ -53,6 +59,14 @@ TSDLLEXPORT char *ts_guc_license_key = TS_DEFAULT_LICENSE;
 char *ts_last_tune_time = NULL;
 char *ts_last_tune_version = NULL;
 char *ts_telemetry_cloud = NULL;
+TSDLLEXPORT bool ts_guc_enable_2pc;
+TSDLLEXPORT int ts_guc_max_insert_batch_size = 1000;
+TSDLLEXPORT bool ts_guc_enable_connection_binary_data;
+TSDLLEXPORT bool ts_guc_enable_client_ddl_on_data_nodes = false;
+TSDLLEXPORT char *ts_guc_ssl_dir = NULL;
+TSDLLEXPORT char *ts_guc_passfile = NULL;
+TSDLLEXPORT bool ts_guc_enable_remote_explain = false;
+TSDLLEXPORT DataFetcherType ts_guc_remote_data_fetcher = RowByRowFetcherType;
```

Comment:
Can `TSDLLEXPORT` be removed? E.g., like in https://github.com/timescale/timescaledb/pull/1923/files#diff-06dd7632738d78ff0f0b35a64ae2702aR52